tools/acle_template.tex

```diff
@@ -24,16 +24,13 @@
 \usepackage[defaultsans,scale=0.95]{lato}
 \usepackage{inconsolata}
 \renewcommand*\familydefault{\sfdefault}
-
-$if(geometry)$
-\usepackage[$for(geometry)$$geometry$$sep$,$endfor$]{geometry}
-$endif$
-
+\usepackage{graphicx}
+\usepackage[$if(landscape)$landscape,$endif$top=3.7cm,bottom=2.7cm,left=3cm,right=2cm,headsep=1.5cm,footskip=.5cm]{geometry}
```

Comment:
This doesn't work well for the tables of the intrinsics :(

If you check, for example, the specs for MVE, you'll see at page 262 that the text overlaps across the columns: 
<img width="842" alt="Screenshot 2021-11-11 at 19 27 57" src="https://user-images.githubusercontent.com/25690309/141357136-8e2b0bcd-e260-4682-9e22-98b160d3f56e.png">

Now, I *think* that to overcome this issue we can insert raw latex code that forces the pages with the tables of the intrinsisc to have a different format. in the following example (inspired from https://tex.stackexchange.com/questions/78278/how-to-set-page-geometry-for-a-single-page-only), we can force the landscape format with narrower left/right margins just from the page that start the intrinsic list, and get back to the portrait format once we are out of the intrinsic list:

```
\newgeometry{landscape, top=3.7cm,bottom=2.7cm,left=1cm,right=1cm,headsep=1.5cm,footskip=.5cm}
# List of Intrinsics

------------------------------------------------------------------------
{intrinsic_table}
```

Of course we need to make sure that the latex command does not get printed out by jekyll... and for that I think we can use the following: https://stackoverflow.com/questions/59584595/ignore-a-line-in-a-markdown-file-with-jekyll

```
{% comment %} 

\newgeometry{landscape, top=3.7cm,bottom=2.7cm,left=1cm,right=1cm,headsep=1.5cm,footskip=.5cm}

{% endcomment %} 
# List of Intrinsics

------------------------------------------------------------------------
{intrinsic_table}
```

Reply:
Right... we also need to make sure the comment/endcomment tags don't get processed by pandoc... more `sed`?

Reply:
If we're using sed anyway why not just replace the section header with the geometry command then section header? Also, the python script doesn't like single curly brackets since it mistakes them for a prompt for a value in the template.